bom/pom.xml

```diff
@@ -139,20 +195,136 @@
+                <artifactId>quarkus-cxf-rt-transports-http-hc5-deployment</artifactId>
+                <version>${quarkus-cxf.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkiverse.cxf</groupId>
+                <artifactId>quarkus-cxf-rt-ws-security</artifactId>
+                <version>${quarkus-cxf.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkiverse.cxf</groupId>
+                <artifactId>quarkus-cxf-rt-ws-security-deployment</artifactId>
+                <version>${quarkus-cxf.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.fasterxml.woodstox</groupId>
+                <artifactId>woodstox-core</artifactId>
+                <version>${woodstox-core.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.sun.xml.messaging.saaj</groupId>
+                <artifactId>saaj-impl</artifactId>
+                <version>${saaj-impl.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.neethi</groupId>
+                <artifactId>neethi</artifactId>
+                <version>${neethi.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.santuario</groupId>
+                <artifactId>xmlsec</artifactId>
+                <version>${xmlsec.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.ehcache</groupId>
+                <artifactId>ehcache</artifactId>
+                <version>${ehcache.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jvnet.mimepull</groupId>
+                <artifactId>mimepull</artifactId>
+                <version>${mimepull.version}</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.l2x6.cq</groupId>
+                <artifactId>cq-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>flatten-bom</id>
+                        <goals>
+                            <goal>flatten-bom</goal>
+                        </goals>
+                        <phase>process-resources</phase>
+                        <configuration>
+                            <originExcludes>
+                                <!-- Remove BOM entries from the flattened BOM which come from quarkus-bom -->
+                                <!-- This is mainly to get rid of potentially outdated quarkus-bom imports brought -->
+                                <!-- via other participant's BOMs that we are importing -->
+                                <originExclude>io.quarkus:quarkus-bom</originExclude>
```

Comment:
How does this work? I mean with this PR, quarkus-cxf-bom does not import quarkus-bom anymore; how does the plugin know which version of quarkus-bom to consult?
I surely missing something.

Reply:
Good catch, `<originExclude>` actually does not need to be here, because quarkus-cxf-bom does not import any third party BOMs that would transitively import quarkus-bom. I left it there, because the plugin is buggy when it is not there. Let me change the comment.

Reply:
Comment changed and https://github.com/l2x6/cq-maven-plugin/issues/82 filed.